api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentStatusController.java

```diff
@@ -189,6 +189,28 @@ public PaymentFailureReportResponse retrievePaymentFailureReport(@RequestParam("
         return new PaymentFailureReportResponse(paymentFailureReportDto);
     }
 
+
+    @Operation(summary = "API to generate report for telephony payments ", description = "Get list of telephony payments by providing date range. MM/dd/yyyy is  the supported date/time format.")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Report Generated"),
+        @ApiResponse(responseCode = "404", description = "No Data found to generate Report"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    @GetMapping("/telephony-payments/telephony-payments-report")
+    public TelephonyPaymentsReportResponse retrieveTelephonyPaymentsReport(@RequestParam("date_from") Date fromDate,
+                                                                            @RequestParam("date_to") Date toDate,
+                                                                            @RequestHeader(required = false) MultiValueMap<String, String> headers,
+                                                                            @RequestHeader("Authorization") String authorization) {
+
+        if (featureToggler.getBooleanValue(PAYMENT_STATUS_UPDATE_FLAG,false)) {
```

Comment:
I think this can be removed. PAYMENT_STATUS_UPDATE_FLAG was intrduced when the payment-failures functionality was added and this just happen to included the corresponding payment-failures report. But this has nothing to do with the Telephony report.